src/shell/shell.cpp

```diff
@@ -1258,6 +1260,31 @@ void SHELL_Init() {
 	        " Volume in drive %c is %s\n"
 	        " Volume Serial Number is %04X-%04X\n"
 	        "\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP",
+	        "Moves files and renames files and directories.\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP_LONG",
+	        "Usage:\n"
+	        "  [color=green]move[reset] [color=white]FILENAME1,FILENAME2[,...][reset] [color=cyan]DESTINATION[reset]\n"
```

Comment:
`[color=white]FILENAME1,FILENAME2[,...][reset]` - shouldn't it rather be `[color=white]FILENAME1[,FILENAME2,...][reset]`?

Reply:
Sure, just changed it.